### ISSUE TITLE
discord.com

### ISSUE THREAD

**Issue by Sonic5435325** [AdguardTeam/AdguardFilters#154930]
### Prerequisites

- [X] This site DOES NOT contain sexually explicit material, otherwise use [NSFW-specific form](./new?template=bug_report_NSFW.yml);
- [X] Filters were updated before reproducing an issue;
- [X] AdGuard product version is up-to-date;
- [X] Browser version is up-to-date;
- [X] If the site or app is broken, disabling AdGuard protection resolves an issue.

### What product do you use?

Other ad blocker

### AdGuard version

uBlock Origin v1.50.1b6

### What type of problem have you encountered?

Annoyances — pop-ups, cookie warnings, etc.

### Which browser(s) do you use?

Firefox

### Which device do you use?

Desktop

### Where is the problem encountered?

https://discord.com/

### What filters do you have enabled?

AdGuard Cookie Notices filter, EasyList Cookie List

### Add your comment and screenshots

1. <comment>
Cooke settings button at the bottom of the page.
Possible solution:
`discord.com##a.navLink-33dQx3:has-text(Cookie Settings)`
2. Screenshots
<details><summary>Screenshot 1:</summary>

![Screenshot from 2023-06-28 12-58-47](https://github.com/AdguardTeam/AdguardFilters/assets/89158249/967e5aa0-3f26-417a-947f-1915f94146da)

</details>

<details><summary>Screenshot N:</summary>

</details>

### Privacy

- [X] I agree to follow this condition

**Comment by DandelionSprout** [comment]
I'm unsure if cookie settings links are what anti-'cookie prompt' lists are meant to block, though the AdGuard team may or may not look at it differently from me.

**Comment by Sonic5435325** [comment]
@DandelionSprout There was a commit which hid one of them here: https://github.com/AdguardTeam/AdguardFilters/issues/154764

They have no purpose/function without cookie popups/scripts. EasyList Cookie also hides these with filters like `##.optanon-toggle-display` (An example of where a filter like this functions: `https://www.thesun.co.uk/`)

**Comment by jellizaveta** [comment]
Agreed with @DandelionSprout . According to the [filtering policy](https://adguard.com/kb/general/ad-filtering/filter-policy/#annoyances-filter). The annoying elements filter blocks obstructions on the page. These elements block the view and make it difficult to see the actual content of the site.
